lib/mapping/MapFeaturesH3M.cpp

```diff
@@ -115,10 +115,11 @@ MapFormatFeaturesH3M MapFormatFeaturesH3M::getFeaturesWOG()
 
 MapFormatFeaturesH3M MapFormatFeaturesH3M::getFeaturesHOTA(uint32_t hotaVersion)

```

Comment:
1. Keep 0,1,2,3 ... but rename hotaVersion to hotaMapVersion.
2. keep hotaVersion but use enum with v0 = 0 , v1, v2, v16 = 3. (v0, v1 since we do not know from what version they started)